### ISSUE TITLE
Release Spirit/Spawn problem

### ISSUE THREAD

**Issue by geeorgee** [HellscreamWoW/Tracker#2024]
Hey guys!

Friend of mine has a problem in-game in N. Barrens zone. He was doing quests near Wailing Caverns and he dies. When he click on Release Spirit this happends :

Screenshot: http://i.imgur.com/C7NiBH2.jpg , http://i.imgur.com/g34PCYT.jpg

**Comment by Malae1** [comment]
Looks like he resurrected fine(aside from being in the air)?

**Comment by geeorgee** [comment]
Yes. ´that´s the problem only way how to get off from the air is by using Hearthstone. He can´t move in air he is stuck

**Comment by geeorgee** [comment]
Everything is O.K. now he is no longer stuck.

**Comment by Malae1** [comment]
Thank you for getting back, I tried to reach you earlier but you weren't online at the time.
